src/components/bcros/businessDetails/Links.vue

```diff
@@ -40,6 +41,14 @@ const isChangeBusinessInfoDisabled = computed(() => {
   return !isAllowed
 })
 
+const showCommentDialog = (show?: boolean) => {
+  if (typeof show !== 'boolean') {
```

Comment:
Why this check?  Do we not want to provided data be boolean and just refuse to have reasonable behavior otherwise ?
If you want to act on other types too, I would say 0 is hide, rather then use 0 to show comment box.
Also, with this, undefined, null, etc... will also display the comment box, no ?
Maybe write it as this following:`isCommentOpen.value = !!show`
This allows falsy/truthy standard behavior from JS

Reply:
This was done for consistency with HeaderActions and how I had to do it there. That said I've made the change to require boolean value (and added true where it was unspecified before) and made the change here and in the other file that was mimicing this (HeaderActions is still like this)